### PR TITLE
Add a critical section to describe a known limitation of powershell w…

### PR DESCRIPTION
**Pull request by paulegradie**
This new critical block (which perhaps could be a warning block) explains a known limitation of Powershell when creating virtual directories.

This addresses a [support issue](https://github.com/OctopusDeploy/Issues/issues/6586) we received which was unable to be resolved directly. Instead, we provide a simple workaround. 

This issue seems to be rare (or if not, most are able to solve it), so we leave documentation that supports users in this case.

### CODE REVIEW

**Review by johnsimons — APPROVED** [review]

File: docs/deployment-examples/iis-websites-and-application-pools.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ You need to fill out the following fields for an IIS Virtual Directory deploymen
 The Virtual Path and Physical Path do not need to match which is one of the true benefits of IIS. You can create a virtual mapping from a URL to a completely unrelated physical path on disk. See [below](/docs/deployment-examples/iis-websites-and-application-pools.md) for more details.
 :::
 
+:::critical
```
Comment (wordlee) [inline docs/deployment-examples/iis-websites-and-application-pools.md:56]: Unfortunately, we only support four types of callous: https://style.octopus.com/markdown#call-outs I've changed this to a warning
```suggestion
:::warning
```

File: docs/deployment-examples/iis-websites-and-application-pools.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ You need to fill out the following fields for an IIS Virtual Directory deploymen
 The Virtual Path and Physical Path do not need to match which is one of the true benefits of IIS. You can create a virtual mapping from a URL to a completely unrelated physical path on disk. See [below](/docs/deployment-examples/iis-websites-and-application-pools.md) for more details.
 :::
 
+:::warning
+We use Powershell to create virtual and physical directories. There is a known limitation with Powerscript which prevents the creation of virtual directories with a leading dot directly under the parent website in IIS. There are two workarounds for this. First, you can manually create a virtually directory on the server using the IIS manager. Alternatively, you can create a physical directory with the same name as your virtual directory target physical directory where your site or application will be installed. For example, you might create a physical directory in the website installation directory called `.well-known`, and then configure your IIS deployment step to create a virtual directory directly under the parent website directory. This issue has been documented [here](https://github.com/OctopusDeploy/Issues/issues/6586).
```
Comment (wordlee) [inline docs/deployment-examples/iis-websites-and-application-pools.md:57]: PowerShell.
```suggestion
We use PowerShell to create virtual and physical directories. There is a known limitation with Powerscript which prevents the creation of virtual directories with a leading dot directly under the parent website in IIS. There are two workarounds for this. First, you can manually create a virtual directory on the server using the IIS manager. Alternatively, you can create a physical directory with the same name as your virtual directory target physical directory where your site or application will be installed. For example, you might create a physical directory in the website installation directory called `.well-known`, and then configure your IIS deployment step to create a virtual directory directly under the parent website directory. This issue has been documented [here](https://github.com/OctopusDeploy/Issues/issues/6586).
```

File: docs/deployment-examples/iis-websites-and-application-pools.md [hunk 3]
```diff
@@ -53,6 +53,10 @@ You need to fill out the following fields for an IIS Virtual Directory deploymen
 The Virtual Path and Physical Path do not need to match which is one of the true benefits of IIS. You can create a virtual mapping from a URL to a completely unrelated physical path on disk. See [below](/docs/deployment-examples/iis-websites-and-application-pools.md) for more details.
 :::
 
+:::warning
+We use PowerShell to create virtual and physical directories. There is a known limitation with Powerscript which prevents the creation of virtual directories with a leading dot directly under the parent website in IIS. There are two workarounds for this. First, you can manually create a virtual directory on the server using the IIS manager. Alternatively, you can create a physical directory with the same name as your virtual directory target physical directory where your site or application will be installed. For example, you might create a physical directory in the website installation directory called `.well-known`, and then configure your IIS deployment step to create a virtual directory directly under the parent website directory. This issue has been documented [here](https://github.com/OctopusDeploy/Issues/issues/6586).
```
Comment (wordlee) [inline docs/deployment-examples/iis-websites-and-application-pools.md:57]: I'm finding `virtual directory target physical directory` a lot to grok, does adding the possessive `s to virtual directory's make sense?
Also, I update Powerscript to PowerShell (please change back if that was intended :))
```suggestion
We use PowerShell to create virtual and physical directories. There is a known limitation with PowerShell which prevents the creation of virtual directories with a leading dot directly under the parent website in IIS. There are two workarounds for this. First, you can manually create a virtual directory on the server using the IIS manager. Alternatively, you can create a physical directory with the same name as your virtual directory's target physical directory where your site or application will be installed. For example, you might create a physical directory in the website installation directory called `.well-known`, and then configure your IIS deployment step to create a virtual directory directly under the parent website directory. This issue has been documented [here](https://github.com/OctopusDeploy/Issues/issues/6586).
```

**Review by wordlee — APPROVED** [review]
Approved with a question about clarity in the body of the PR.

File: docs/deployment-examples/iis-websites-and-application-pools.md (hunk 3 above)
Comment (paulegradie) [reply docs/deployment-examples/iis-websites-and-application-pools.md:57]: XD Powerscript omg  (I'm working with typescript a lot atm I think).

Yeah, I think you're right. I'll commit these changes inline here.

**Pull request merged**